isamples_metadata/isamplesfasttext.py

```diff
@@ -0,0 +1,44 @@
+# Adapted from https://github.com/isamplesorg/vocabulary_learning/blob/develop/fasttext_interface/DwC_predict.py
+
+import fasttext
+import os.path
+import typing
+import re
+
+from fasttext.FastText import _FastText
+
+MODEL_FILENAME = "./sampledFeature.bin"
+THIS_PATH = os.path.dirname(os.path.abspath(__file__))
+MODEL_PATH = os.path.join(THIS_PATH, MODEL_FILENAME)
+MODEL = None
+RESULT_PREFIX = "__label__"
+
+
+def _model() -> _FastText:
+    global MODEL
+    if MODEL is None:
+        MODEL = fasttext.load_model(MODEL_PATH)
+    return MODEL
+
+
+def predict_sampled_feature(context: typing.List[typing.AnyStr]) -> typing.AnyStr:
+    """
+    Invoke the pre-trained fasttext model to predict the sampledFeature label for the specified string inputs
+    :param context: The unprocessed inputs to the model, taken from a source record
+    :return: String label to be used as the sampled feature for the source record
+    """
+
```

Comment:
@qgan7125 How does this look to you?  I tried to repackage your module given what we have at our disposal when we'll invoke it, then reuse the same processing logic.  Do this look right to you?  It "works" but it's difficult to say exactly whether it's correct or not.

Reply:
Yeah, I think it's absolutely correct! I did want to ask you whether I need to write some interfaces as same as your current work this morning but I forgot to ask.